2.0/SciFiHelmet/README.md

```diff
@@ -0,0 +1,11 @@
+# Sci Fi Helmet
+## Screenshot
+
+![screenshot](screenshot/screenshot.jpg)
+
+
+## License Information
+
+["Sci Fi Helmet" by Michael Pavlovic](http://quixel.se/usermanual/quixelsuite/doku.php?id=ddo_samples)
```

Comment:
On the website. the license is the following "CC Attribution-Share Alike 3.0 Unported"
https://creativecommons.org/licenses/by-sa/3.0/